src/sentences.rs

```diff
@@ -46,13 +55,19 @@ fn create_sentence<'r>(
     let rows = match result {
         Ok(rows) => rows,
         Err(ref e) => {
-            if e.code() == Some(&UNIQUE_VIOLATION) {
 
-                return  Response::build()
+            let error = e.code();
+            if error == Some(&UNIQUE_VIOLATION) {
+                return Response::build()
                     .status(Status::Conflict)
-                    .finalize()
-                ;
+                    .finalize();
             }
+            else if error == Some(&FOREIGN_KEY_VIOLATION) {
```

Comment:
no need for a else as you have a return in the if

Reply:
You're right. Done.